lib/reports/ReportCollisionDirectory.js

```diff
@@ -69,21 +69,112 @@ class ReportCollisionDirectory extends ReportBaseCrash {
      * TODO: this violates the "no shared references" invariant in `ReportBase#transformData`.
```

Comment:
get rid of this comment - now that we've removed `rawCollisions` here, it's no longer true!